packages/foreman/rubygem-mail/rubygem-mail.spec

```diff
@@ -16,6 +16,10 @@ BuildRequires: rubygems-devel
 BuildArch: noarch
 # end specfile generated dependencies
 
+# Prefer to consume net-smtp/net-imap/net-pop as a default gem
```

Comment:
There are no `Provides: bundled(rubygem-net-*)`, so I didn't add any `(Build)Requires` on that like we do for `racc`.